build.gradle

```diff
@@ -32,7 +32,7 @@ apply plugin: 'io.spring.dependency-management'
 
 def versions = [
         junit              : '5.8.2',
-        lombok             : '1.18.6',
+        lombok             : '1.18.22',
         gradlePitest       : '1.3.0',
```

Comment:
Please check PITest runs locally `./gradlew pitest`

Reply:
thanks for review comment ,need rework on this .